src/providers/WalletProvider.tsx

```diff
@@ -23,28 +24,36 @@ export const WalletProvider = ({ children }: { children: React.ReactNode }) => {
   const [network, setNetwork] = useState<string>();
   const [networkPassphrase, setNetworkPassphrase] = useState<string>();
   const [isPending, startTransition] = useTransition();
+  const popupLock = useRef(false);
 
   const nullify = () => {
     setNetwork(undefined);
     setNetworkPassphrase(undefined);
     setAddress(undefined);
+    storage.setItem("walletId", "");
```

Comment:
Do we really even need to track this in local storage anymore? It looks like there's already an array being tracked by stellar-wallets-kit. They might be handling preferred connections internally, and since we removed that value from `StellarWalletsKit()` instantiation, we might be able to remove ours entirely.